src/main/java/AppMain.java

```diff
@@ -0,0 +1,31 @@
+import logic.BouquetLogic;
+import model.Bouquet;
+import model.DecorationPlant;
+import model.Flower;
+import view.BouquetPrinter;
+
+import java.util.List;
+
+
+public class AppMain {
+    public static void main(String[] args) {
+
+        FlowerCreator flowerCreator = new FlowerCreator();
```

Comment:
I think one Bouquet would be enough that create Bouquet  with different types of flowers